### PR TITLE
fix grammar

### PR DESCRIPTION
**Pull request by Laplacence**

### CODE REVIEW

**Comment by aykutkardas** [comment]
What do you think of these fixes? @isPoto

File: src/localization/zh-cn/learn.json [hunk 1]
```diff
@@ -78,13 +78,13 @@
   "examples.questionMark.description": "请编写表达式，用 `?` 表示字母 `n` 在文本中是可选的，使 `a` 和 `an` 都可以匹配到。",
 
   "examples.quantifier.title": "练习：大括号 - 1",
-  "examples.quantifier.description": "用 `{}` 编写表达式，匹配文本中的 `4` 位阿拉伯数字。",
+  "examples.quantifier.description": "用 `{}` 编写表达式，匹配文本中 `4` 位阿拉伯数字。",
 
   "examples.quantifierMin.title": "练习：大括号 - 2",
-  "examples.quantifierMin.description": "用 `{}` 编写表达式，匹配文本中位数至少为 `2` 的阿拉伯数字。",
```
Comment (isPoto) [inline src/localization/zh-cn/learn.json:84]: 匹配文本中，位数至少为 `2` 的阿拉伯数字

File: src/localization/zh-cn/learn.json [hunk 2]
```diff
@@ -78,13 +78,13 @@
...
+  "examples.quantifierMin.description": "用 `{}` 编写表达式，匹配文本中至少为 `2` 位阿拉伯数字。",
 
   "examples.quantifierRange.title": "练习：大括号 - 3",
-  "examples.quantifierRange.description": "用 `{}` 编写表达式，匹配文本中位数为 `1` 至 `4` 的阿拉伯数字。",
```
Comment (isPoto) [inline src/localization/zh-cn/learn.json:87]: 匹配文本中，位数为 `1` 至 `4` 的阿拉伯数字

**Review by isPoto — CHANGES_REQUESTED** [review]
I have made some improvements for better grammatical understanding. @aykutkardas

File: src/localization/zh-cn/learn.json (hunk 1 above)
Comment (Laplacence) [reply src/localization/zh-cn/learn.json:84]: 这一部分为`{}`的使用练习，去掉会不会产生一些歧义？

File: src/localization/zh-cn/learn.json [hunk 3]
```diff
@@ -78,13 +78,13 @@
   "examples.questionMark.description": "请编写表达式，用 `?` 表示字母 `n` 在文本中是可选的，使 `a` 和 `an` 都可以匹配到。",
 
   "examples.quantifier.title": "练习：大括号 - 1",
-  "examples.quantifier.description": "用 `{}` 编写表达式，匹配文本中的 `4` 位阿拉伯数字。",
+  "examples.quantifier.description": "用 `{}` 编写表达式，匹配文本中 `4` 位阿拉伯数字。",
 
   "examples.quantifierMin.title": "练习：大括号 - 2",
-  "examples.quantifierMin.description": "用 `{}` 编写表达式，匹配文本中位数至少为 `2` 的阿拉伯数字。",
+  "examples.quantifierMin.description": "用 `{}` 编写表达式，匹配文本中至少为 `2` 位阿拉伯数字。",
```
Comment (isPoto) [inline src/localization/zh-cn/learn.json:84]:
```suggestion
  "examples.quantifierMin.description": "用 `{}` 编写表达式，匹配文本中，位数至少为 `2` 的阿拉伯数字。",
```

File: src/localization/zh-cn/learn.json [hunk 4]
```diff
@@ -78,13 +78,13 @@
...
 
   "examples.quantifierRange.title": "练习：大括号 - 3",
-  "examples.quantifierRange.description": "用 `{}` 编写表达式，匹配文本中位数为 `1` 至 `4` 的阿拉伯数字。",
+  "examples.quantifierRange.description": "用 `{}` 编写表达式，匹配文本中 `1` 至 `4` 位阿拉伯数字。",
```
Comment (isPoto) [inline src/localization/zh-cn/learn.json:87]:
```suggestion
  "examples.quantifierRange.description": "用 `{}` 编写表达式，匹配文本中，位数为 `1` 至 `4` 的阿拉伯数字。",
```

File: src/localization/zh-cn/learn.json (hunk 1 above)
Comment (Laplacence) [reply src/localization/zh-cn/learn.json:84]: 个人感觉，位数单独拿出来放在前面有点不太适合中文语境。

**Review by isPoto — CHANGES_REQUESTED** [review]
My IT friend suggested me make a new copy of Contributor that is easy for you to adopt

File: src/localization/zh-cn/learn.json (hunk 1 above)
Comment (shugen002) [reply src/localization/zh-cn/learn.json:84]: 确实，一般很少会说位数。

File: src/localization/zh-cn/learn.json [hunk 5]
```diff
@@ -78,13 +78,13 @@
   "examples.questionMark.description": "请编写表达式，用 `?` 表示字母 `n` 在文本中是可选的，使 `a` 和 `an` 都可以匹配到。",
 
   "examples.quantifier.title": "练习：大括号 - 1",
-  "examples.quantifier.description": "用 `{}` 编写表达式，匹配文本中的 `4` 位阿拉伯数字。",
```
Comment (isPoto) [inline src/localization/zh-cn/learn.json:81]: Agree to the changes

**Review by isPoto — COMMENTED** [review]
Agreed on one change, plus there's a discussion on translation, and I look forward to a better translated version being submitted.

File: src/localization/zh-cn/learn.json (hunk 5 above)
Comment (Laplacence) [reply src/localization/zh-cn/learn.json:81]: LGTM

**Comment by bxb100** [comment]
BTW, why not using `个`
example: `a{3,}` explain: "至少三个 a"

**Comment by Laplacence** [comment]
@bxb100 
如果可以使用‘位’这样的量词，那我们就尽量不使用‘个’这样通用的量词。

**Comment by isPoto** [comment]
https://zh.wikipedia.org/wiki/%E6%95%B0%E4%BD%8D_(%E6%95%B0%E5%AD%A6)

According to the common Chinese terminology for numbers, `位' should be used instead of `个'

Considering that some users have urged me to fix the errors mentioned in this version of the PR, I am fixing and launching the PR first. i want to fix the obvious errors first, and as for the translation wording, I want to put aside the controversy. It is more important for users to experience the correct translation as soon as possible

**Comment by aykutkardas** [comment]
Thanks for your contribution. @Laplacence 

I've accepted @isPoto 's pull request for now. Please keep reviewing and making improvements to the translation.

**Pull request closed without merging**